src/sentry/integrations/slack/integration.py

```diff
@@ -234,6 +250,8 @@ def dispatch(self, request, pipeline):
                 template="sentry/integrations/slack-reauth-introduction.html",
                 context={
                     "next_url": "%s%s" % (absolute_uri("/extensions/slack/setup/"), next_param),
+                    "workspace": integration.name,
+                    "extra_orgs": extra_orgs.values(),
```

Comment:
@MeredithAnya Wondering if it makes sense to test this somehow

Reply:
@scefali like in an acceptance test?

Reply:
@MeredithAnya More like a mock that tests what was passed as the context. Or to test the new state in the pipeline. Whatever is easier.